docs/design/2022-03-03-rc-read-tso-optimization.md

```diff
@@ -0,0 +1,36 @@
+# Read-Consistency Read With Tso Check
+
+- Author(s): cfzjywxk
+- Last updated: March. 3, 2022
+- Discussion at:
+
+## Motivation
+
+For the read-consistency isolation level read requests in a single transaction, each will need to fetch a new `tso` to read the latest committed data.
+If the workload is a read-heavy one or the read `qps` is large, fetching tso each time will increase the query lantecy.
```

Comment:
Why the words such as "tso" and "qps" are in inline code format (in backquotes)?